engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -525,29 +525,21 @@ def short_name(self):
             )
 
     @property
-    def short_name_with_maintenance_status(self):
```

Comment:
this is not referenced anywhere ([repo code search](https://github.com/search?q=repo:grafana/oncall%20short_name_with_maintenance_status&type=code))